src/components/content-tab-assistant.tsx

```diff
@@ -158,11 +159,19 @@ export const Message = ( { children, isUser, className }: MessageProps ) => {
 			) }
 		>
 			<div
+				id={ id }
+				role="group"
+				aria-labelledby={ id }
 				className={ cx(
 					'inline-block p-3 rounded border border-gray-300 lg:max-w-[70%] select-text',
 					! isUser ? 'bg-white' : 'bg-white/45'
 				) }
 			>
+				<div className="relative">
+					<span className="sr-only">
+						{ isUser ? __( 'Your message' ) : __( 'Studio Assistant' ) },
```

Comment:
I wonder if to increase comprehensibility we could say "Studio Assistant's message", instead of Studio Assistant. This will be aligned with the other text, "Your message". WDY?T

Reply:
Good thought. I selected the current strings after reviewing and drawing inspiration from Apple's macOS Messages app. 

One positive I perceive for _not_ appending "message" to each entry is that is prioritizes brevity, which could be beneficial when navigating a lot of message text. The use of "message" for the labeling non-assistant messages feels somewhat necessary as we may not always have the user's name. 

Much like most a11y design, I recognize the comprehensibility of the strings are fairly subjective. I do not have a strong conviction one way or the other. 

Which approach would you like to take?

Reply:
@fluiddot I merged these changes as-is to retain momentum, but I am happy to follow up on this discussion in a new pull request. Please feel free to share your thoughts whenever it's convenient.

Reply:
> One positive I perceive for _not_ appending "message" to each entry is that is prioritizes brevity, which could be beneficial when navigating a lot of message text. The use of "message" for the labeling non-assistant messages feels somewhat necessary as we may not always have the user's name.

Good point. Brevity is an important factor here.

> Much like most a11y design, I recognize the comprehensibility of the strings are fairly subjective. I do not have a strong conviction one way or the other.
> Which approach would you like to take?

Yeah. From my side, I don't have a strong opinion on which one to use, so we can keep the current strings 👍 .